src/_posts/aframe-v0.7.0.md

```diff
@@ -65,4 +65,4 @@ Since v0.6.0 was released in Jun 29 2017:
 
 ## What's Next?
 
-We will keep profiling and measuring to make sure that A-Frame overhead is negligable and your content runs butter smooth. We are starting to consolidate the API and want to get soon to a point we feel confortable with the design and ship a 1.0.0 release. 0.7.0 has shippped a bit earlier that we planned to make sure that Windows Mixed Reality headsets and Microsoft Edge are well supported at launch. A couple of features like vive tracker support and spectator camera have moved to 0.8.0. Keep an eye on the [roadmap](https://github.com/aframevr/aframe/blob/master/ROADMAP.md) to see what's coming.
+We will keep profiling and measuring to make sure that A-Frame overhead is negligable and your content runs buttery smooth. We are starting to consolidate the API and want to get soon to a point we feel confortable with the design and ship a 1.0.0 release. 0.7.0 has shippped a bit earlier than we planned to make sure that Windows Mixed Reality headsets and Microsoft Edge are well supported at launch. A couple of features, such as Vive tracker support and spectator camera, have moved to 0.8.0. Keep an eye on the [roadmap](https://github.com/aframevr/aframe/blob/master/ROADMAP.md) to see what's coming.
```

Comment:
btw, check out GitHub's [Unified view](https://github.com/aframevr/aframe-site/pull/464/files?short_path=388328d#diff-388328d24f5740278d430db26c9f8718) to see the diff (instead of the Split view, which doesn't show the diff for this single-line change)